packages/hunspell_en_us.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Hunspell_en_us < Package
+  description 'Hunspell US English dictionary'
+  homepage 'http://hunspell.github.io/'
+  version '1.7.0'
+  source_url 'https://github.com/hunspell/hunspell/archive/v1.7.0.tar.gz'
+  source_sha256 'bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'hunspell'
+
+  def self.install
+    system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
+    system 'mkdir -p $HOME/Library/Spelling'
```

Comment:
This won't work in single quotes.  In any event, this line is not needed since the commands below already handle the directory.